test/games/strategy/engine/framework/map/download/DownloadFileParserTest.java

```diff
@@ -22,53 +19,53 @@
   public void testParseMap() {
     final ByteArrayInputStream inputStream = new ByteArrayInputStream(buildTestXml());
     List<DownloadFileDescription> games = DownloadFileParser.parse(inputStream);
-    assertThat(games.size(), is(4));
+    assertEquals(4, games.size());
     DownloadFileDescription desc = games.get(0);
-    assertThat(desc.getUrl(), is("http://example.com/games/game.zip"));
-    assertThat(desc.getDescription(), Matchers.containsString("Some notes"));
-    assertThat(desc.getMapName(), is(GAME_NAME));
+    assertEquals(desc.getUrl(), "http://example.com/games/game.zip");
```

Comment:
The question I'm asking myself is, should I revert my changes or leave them as is